pkg/multitenant/mtinfopb/info.proto

```diff
@@ -81,4 +89,28 @@ message TenantInfoWithUsage {
   }
   optional TenantInfo info = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
   optional Usage usage = 2;
+
+  // ExtraColumns contain the additional metadata from the other
+  // columns in system.tenants not otherwise encoded in the info field.
+  message ExtraColumns {
```

Comment:
I'm not sure I immediately see what we get by nesting these specific cols in this `ExtraColumns` message vs just having name, id, etc be fields of `TenantInfoWithUsage` (and renaming it to TenantInfoExtended)?